### DISCUSSION TITLE
WasmPlugin source code

### DISCUSSION THREAD

**Discussion by avistech** [istio/istio#47036] [General]
Hi there, do we have access to WasmPlugin source code that is mention in the example code @ [https://istio.io/latest/docs/tasks/observability/metrics/classify-metrics/](https://istio.io/latest/docs/tasks/observability/metrics/classify-metrics/). we want to know more about match condition, what should be condition for fallback option of always "true".

https://storage.googleapis.com/istio-build/proxy/attributegen-359dcd3a19f109c50e97517fe6b1e2676e870c4d.wasm

**Comment by zirain** [comment]
it's here now:https://github.com/istio-ecosystem/wasm-extensions/tree/master/extensions
